Rakefile

```diff
@@ -60,3 +59,23 @@ end
 task :spec_postgres do
   run_spec.call('postgres')
 end
+
+task :spec_integration do
+  def port_available?(port)
+    server = TCPServer.open("0.0.0.0", port)
+    server.close
+    true
+  rescue Errno::EADDRINUSE
+    false
+  end
+
+  begin
+    ENV['MESSAGE_BUS_BACKEND'] = 'memory'
```

Comment:
Is it possible to also run these against other backends?

Reply:
It is possible but I don't think we need to do so though because I'm only interested in testing the HTTP client. The backend is configured here https://github.com/SamSaffron/message_bus/pull/192/files#diff-926cc5ae85da76b1b9c6ba536922080eR3.